code/game/machinery/quantum_pad.dm

```diff
@@ -102,7 +102,7 @@
 	do_sparks(5, 1, get_turf(src))
 
 /obj/machinery/quantumpad/attack_ghost(mob/dead/observer/ghost)
-	if(linked_pad)
+	if(linked_pad && !QDELETED(linked_pad))
```

Comment:
```suggestion
	if(!QDELETED(linked_pad))
```

Reply:
The other conditionals in the quantum pad code have similar `!linkedpad || !QDELETED(linkedpad)` checks. I'd figured I'd keep both here for readability, but if you think I should remove that part I will

Reply:
```dm
#define QDELETED(X) (!X || QDELING(X))
```
it will already null check it in the define, the `linked_pad` check doesn't do anything here.

Reply:
Yeah, I know. I just mean for readability for like *new* coders